README.md

```diff
@@ -76,7 +76,7 @@ This is a Ruby on Rails application that contains:
 
 ### Changes to the landing page
```

Comment:
Minor: I found this confusing when I read it initially. Could we rename it to "Changes to a Smart Answer"? The process is the same for all parts of the flow.

Reply:
Less minor: having this here did slow me down when writing/merging the initial smart answer, since it's weirdly separate to the "Smart Answer Flow Development" section in the README. 

Could we make it it's own section and link/step instead? Happy for this to be done later.

Reply:
Ah darn all I did was fix the rake task name on that line - I've not even read it 🙈. Your points sound good though. Is this something you wouldn't mind picking up? I think you've probably got a clearer idea than me what it should say.